extensions/github1s/src/helpers/util.ts

```diff
@@ -78,3 +78,12 @@ export const encodeFilePath = (filePath: string): string => {
 		.map((segment) => encodeURIComponent(segment))
 		.join('/');
 };
+
+// parse the querystring simply: 'a=1&b=2&c=3'
+export const parseQuery = (query: string): { [key: string]: string } => {
```

Comment:
I suggest we use https://github.com/ljharb/qs or https://github.com/sindresorhus/query-string to handle the query parsing since there might be problems with escaped character and other edge cases.

Reply:
Thank for you recommendation! I wanted to use qs yesterday, but I found the bundle size of the `qs` looks like too big, so I write one simply. Bow I think `query-string` is better for browsers, I will use it~

Reply:
It's a great point.

I think we should set up a visible indicator of the performance, there are tools like https://github.com/webpack-contrib/webpack-bundle-analyzer that can do the code analysis, but I think it needs some setup to make sure it works with this project.

Another direction is the browser measurement like the lighthouse (I have installed it onVercel) and  https://vercel.com/integrations/debugbear that can do the analysis but it needs to configure an account. Could you please take a look if you are interested & when you have time?

Reply:
> Another direction is the browser measurement like the lighthouse (I have installed it onVercel) and https://vercel.com/integrations/debugbear that can do the analysis but it needs to configure an account. Could you please take a look if you are interested & when you have time?

Thank you so much! I will configurate it next

Reply:
> Another direction is the browser measurement like the lighthouse (I have installed it onVercel) and https://vercel.com/integrations/debugbear that can do the analysis but it needs to configure an account.

@xcv58 The lighthouse is working, I think we can not use the debugbear because it is not for free.😂

Reply:
Yes. Sorry I didn’t know that before. I think we can keep in mind about this and looking for measurements in the long term.